README.md

```diff
@@ -263,6 +271,49 @@ Renders the widget to the DOM, and passes control back to your app through succe
 - `success` *(optional)* - Function that is called when the user has completed an authentication flow. If an [OpenID Connect redirect flow](#openid-connect) is used, this function can be omitted.
 - `error` *(optional)* - Function that is called when the widget has been initialized with invalid config options, or has entered a state it cannot recover from. If omitted, a default function is used to output errors to the console.
 
+**OIDC Configuration:**
+
+This is the recommended configuration for new integrations as it uses a more lightweight REST-based protocol and has more widespread usage. For more information, see developer documentation on [choosing a protocol](https://developer.okta.com/docs/guides/build-sso-integration/openidconnect/overview/#choosing-a-protocol)
+
+```javascript
+signIn.renderEl(
+  // Assumes there is an empty element on the page with an id of 'osw-container'
+  {el: '#osw-container'},
+
+  function success(res) {
+    // Handling of other statuses in `res.status` is the same as the non-OIDC configuration
+
+    // The user has successfully completed the authentication flow
+    if (res.status === 'SUCCESS') {
+
+      // If the widget is configured for OIDC with a single responseType, the
+      // response will be the token.
+      // i.e. authParams.responseType = 'id_token':
+
+      // Example handling of single responseType:
+      // myTokenManager.add('my_id_token', res);
+
+      // If the widget is configured for OIDC with multiple responseTypes, the
+      // response will be an array of tokens:
+      // i.e. authParams.responseType = ['id_token', 'token']
+
+      // Example handling of multiple responseTypes instead:
+      // myTokenManager.add('my_id_token', res[0]);
+      // myTokenManager.add('my_access_token', res[1]);
+
+      return;
+    }
+
+  },
+
+  function error(err) {
+    // Same as non-OIDC error handling function
+  }
+);
+```
+
+**Non-OIDC Configuration:**
+
```

Comment:
Can we provide any text here on WHY a user would choose a non-oidc configuration? What is the use-case?

Reply:
I think this **might be** when using the customized widget that is Okta hosted

Reply:
I swapped the spots of OIDC and non-OIDC but I have added a sentence with an example of when non-OIDC response might be handled (hosted SIW to log into Okta dashboard)